File: user/Presentation/DTOs/PaymentInfoDto.cs
namespace PTManagementSystem.Presentation.DTOs
{
    public class PaymentInfoDto
    {
        public string cardHolder { get; set; } = string.Empty;
        public string cardNumber { get; set; } = string.Empty;
        public string expiryMonth { get; set; } = string.Empty;
        public string expiryYear{ get; set; } = string.Empty;
        public string cvc { get; set; } = string.Empty;
        //public string BillingAddress { get; set; } = string.Empty;
    }
}

File: payment/paymentDemo/src/main/java/com/example/paymentDemo/dto/PaymentInfoDto.java
package com.example.paymentDemo.dto;
import com.fasterxml.jackson.annotation.JsonProperty;

public class PaymentInfoDto {
    @JsonProperty("cardHolder")
    private String cardHolder;
    @JsonProperty("cardNumber") 
    private String cardNumber;
    @JsonProperty("expiryMonth")
    private String expiryMonth;
    @JsonProperty("expiryYear")
    private String expiryYear;
    @JsonProperty("cvc")
    private String cvc;

    public PaymentInfoDto() {
    }

    // Getters and Setters
    public String getCardHolder() {
        return cardHolder;
    }

    public void setCardHolder(String cardHolder) {
        this.cardHolder = cardHolder;
    }

    public String getCardNumber() {
        return cardNumber;
    }

    public void setCardNumber(String cardNumber) {
        this.cardNumber = cardNumber;
    }

    public String getExpiryMonth() {
        return expiryMonth;
    }

    public void setExpiryMonth(String expiryMonth) {
        this.expiryMonth = expiryMonth;
    }

    public String getExpiryYear() {
        return expiryYear;
    }

    public void setExpiryYear(String expiryYear) {
        this.expiryYear = expiryYear;
    }

    public String getCvc() {
        return cvc;
    }

    public void setCvc(String cvc) {
        this.cvc = cvc;
    }
}
